adal/proguard-rules.pro

```diff
@@ -1,11 +1,8 @@
-# To enable ProGuard in your project, edit project.properties
-# to define the proguard.config property as described in that file.
-#
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in ${sdk.dir}/tools/proguard/proguard-android.txt
-# You can edit the include path and order by changing the ProGuard
-# include property in project.properties.
+# in /Users/randallilama/sdk/android-sdk-macosx/tools/proguard/proguard-android.txt
```

Comment:
Can you please change "**/Users/randallilama/sdk/android-sdk-macosx/tools/proguard/proguard-android.txt**" to **${sdk.dir}/tools/proguard/proguard-android.txt**